src/backend/app/projects/project_routes.py

```diff
@@ -942,6 +944,13 @@ async def generate_files(
             },
         )
 
+    if project.custom_tms_url:
+        basemap_in = project_schemas.BasemapGenerate(
+            tile_source="tms", file_format="pmtiles", tms_url=project.custom_tms_url
+        )
+        org_id = project.organisation_id
+        await generate_basemap(project_id, org_id, basemap_in, db, background_tasks)
```

Comment:
Its key that this returns immediately when the background task is submitted - just checking this works well upon testing?

Reply:
Yes this worked upon testing